app/Policies/Store/RegistrationPolicy.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Policies\Store;
+
+use App\CentreUser;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class RegistrationPolicy
+{
+    use HandlesAuthorization;
+
+    // These are permissions that we need to check a user for.
+    public function updateDiary(CentreUser $user)
```

Comment:
I know I've said before but why different named policies that check same thing?

Reply:
At this poin ti'm loathe to change it without good reason.